vllm/spec_decode/interfaces.py

```diff
@@ -55,7 +55,7 @@ def __repr__(self):
 class SpeculativeProposer(ABC):
 
     @abstractmethod
-    def get_proposals(
+    def get_spec_proposals(
```

Comment:
Making this name consistent (in the proposer workers it's called `get_spec_proposals`)